srcpkgs/babashka/template

```diff
@@ -59,7 +59,7 @@ post_extract() {
 }
```

Comment:
This needs change too, to mandrel21

Reply:
pushed that now, but `bb -e '(babashka.process/exec "ls")'` doesn't work still after the revbump.

Reply:
We need to fix that before merging then.

Reply:
When building `babashka`, under `--module-path` it has
```
/usr/lib/jvm/mandrel21/lib/svm-preview/builder/svm-foreign.jar:/usr/lib/jvm/mandrel21/lib/jvmci/word.jar:/usr/lib/jvm/mandrel21/lib/jvmci/nativeimage.jar:/usr/lib/jvm/mandrel21/lib/jvmci/collections.jar:/usr/lib/jvm/mandrel21/lib/truffle/truffle-compiler.jar:/usr/lib/jvm/mandrel21/lib/svm/builder/pointsto.jar:/usr/lib/jvm/mandrel21/lib/svm/builder/svm.jar:/usr/lib/jvm/mandrel21/lib/svm/builder/objectfile.jar:/usr/lib/jvm/mandrel21/lib/svm/builder/native-image-base.jar
```
Isn't that one supposed to have `graal-sdk.jar` as well?

Reply:
I tried patching this in but it didn't help?

Reply:
Here's two build logs for identical `babashka` versions - [bb_old.txt](https://github.com/void-linux/void-packages/files/14076338/bb_old.txt) with mandrel-23.0.1.2 JDK 17, [bb_new.txt](https://github.com/void-linux/void-packages/files/14076337/bb_new.txt) with mandrel-23.1.2.0 JDK 21. Diffing them, the missing `graal-sdk.jar` in the later's `--module-path` should be the issue. Maybe a change in graal/mandrel internal across versions? Dunno how to go about debugging that.

Reply:
So, the difference is that on mandrel 17 this graal-sdk.jar contains lots of things, but on mandrel 21 it just contains some properties.  The actual classes are in the svm*jar things.

Babashka exec needs the class org.graalvm.nativeimage.ProcessProperties to exist, which used to be in graal-sdk.jar but is now in jvmci/nativeimage.jar... which is in module-path already tho.